style.css

```diff
@@ -298,7 +298,7 @@ input:hover {
   border: 1px solid #253858;
 }
```

Comment:
- [ ] To Improve the user interface of your portfolio, I will encourage you to fix the alignment issues with the styling of your project. Using the Figma template as a guide fix the styling on the cards in the work section and language section of your project. Try to make it look exactly like the original template. 

> Your design
<img width="486" alt="Screenshot 2023-05-19 at 13 26 34" src="https://github.com/mariegrace31/portfolio_project/assets/71140133/206dc12f-fe10-4108-8853-2c69cb25615f">

> Figma design
<img width="500" alt="Screenshot 2023-05-19 at 13 27 35" src="https://github.com/mariegrace31/portfolio_project/assets/71140133/a6388a3e-3acc-4a98-94b4-80fb9345ba4b">

> Your design
<img width="929" alt="Screenshot 2023-05-19 at 13 32 24" src="https://github.com/mariegrace31/portfolio_project/assets/71140133/1b80bbbd-edb7-41d8-a5c3-eb7602685138">

> Figma design
<img width="525" alt="Screenshot 2023-05-19 at 13 32 45" src="https://github.com/mariegrace31/portfolio_project/assets/71140133/048ae591-49c8-4fc5-94a8-89c3decc17a7">